shotover-proxy/examples/windsock/main.rs

```diff
@@ -24,31 +24,73 @@ fn main() {
                 CassandraDb::Cassandra,
                 Topology::Single,
                 Shotover::None,
+                Compression::None,
+            )),
+            Box::new(CassandraBench::new(
+                CassandraDb::Cassandra,
+                Topology::Single,
+                Shotover::None,
+                Compression::Lz4,
             )),
             Box::new(CassandraBench::new(
                 CassandraDb::Cassandra,
                 Topology::Single,
                 Shotover::Standard,
+                Compression::None,
+            )),
+            Box::new(CassandraBench::new(
+                CassandraDb::Cassandra,
+                Topology::Single,
+                Shotover::Standard,
+                Compression::Lz4,
+            )),
+            Box::new(CassandraBench::new(
+                CassandraDb::Cassandra,
+                Topology::Cluster3,
+                Shotover::None,
+                Compression::None,
             )),
             Box::new(CassandraBench::new(
                 CassandraDb::Cassandra,
                 Topology::Cluster3,
                 Shotover::None,
+                Compression::Lz4,
             )),
             Box::new(CassandraBench::new(
                 CassandraDb::Cassandra,
                 Topology::Cluster3,
                 Shotover::Standard,
+                Compression::None,
+            )),
+            Box::new(CassandraBench::new(
+                CassandraDb::Cassandra,
+                Topology::Cluster3,
+                Shotover::Standard,
+                Compression::Lz4,
             )),
             Box::new(CassandraBench::new(
                 CassandraDb::Mocked,
                 Topology::Single,
                 Shotover::None,
+                Compression::None,
+            )),
+            Box::new(CassandraBench::new(
+                CassandraDb::Mocked,
+                Topology::Single,
+                Shotover::None,
+                Compression::Lz4,
```

Comment:
I dont think this could work because the mocked cassandra doesnt support compression?